tfhe/benches/core_crypto/pbs_bench.rs

```diff
@@ -35,6 +37,29 @@ const SHORTINT_BENCH_PARAMS: [ClassicPBSParameters; 19] = [
     PARAM_MESSAGE_4_CARRY_4_PBS_KS,
 ];
 
+#[cfg(feature = "gpu")]
+const SHORTINT_BENCH_PARAMS: [ClassicPBSParameters; 2] = [
+    // PARAM_MESSAGE_1_CARRY_0_KS_PBS,
+    PARAM_MESSAGE_1_CARRY_1_KS_PBS,
+    // PARAM_MESSAGE_2_CARRY_0_KS_PBS,
+    // PARAM_MESSAGE_2_CARRY_1_KS_PBS,
+    PARAM_MESSAGE_2_CARRY_2_KS_PBS,
+    // PARAM_MESSAGE_3_CARRY_0_KS_PBS,
+    // PARAM_MESSAGE_3_CARRY_2_KS_PBS,
+    // PARAM_MESSAGE_3_CARRY_3_KS_PBS,
+    // PARAM_MESSAGE_4_CARRY_0_KS_PBS,
```

Comment:
Should we make a more thorough selection by either selecting all of the parameters (thus removing the need to gate the constant), or by selecting only some of them and removing all the commented ones ?

Reply:
I'll remove the gate, so we bench the same parameter sets on GPU & CPU.